recipes/xorg-xft/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set xorg_name = "libXft" %}
```

Comment:
If this is going to be `libXft`, then the directory needs to be `xorg-libxft`. This means the package will also be named `xorg-libxft`. Is the `lib` something you want in there or not? How do others name this package? Part of the reason I ask is this hasn't been converted because of this naming issue. Thoughts @epruesse @pkgw?